client/components/segmented-selection/style.scss

```diff
@@ -17,7 +17,7 @@
 
 .woocommerce-segmented-selection__label {
 	background-color: $core-grey-light-100;
-	padding: 12px 12px 12px 24px;
+	padding: 12px 12px 12px 36px;
```

Comment:
Maybe `padding: $gap-small $gap-small $gap-small $gap-largest;` instead?

Reply:
`$gap-largest` is 40px, so it would be a slightly different style if that was used.